src/ripe_rainbow/interactive/drivers.py

```diff
@@ -125,15 +125,14 @@ def write_text(self, element, text):
         element.send_keys(text)
         return element
 
-    def click(self, element, scroll = True, scroll_sleep = None):
+    def click(self, element, scroll = True):
         from selenium.webdriver.common.action_chains import ActionChains
         from selenium.common.exceptions import ElementClickInterceptedException, ElementNotVisibleException, WebDriverException
 
         try:
             if scroll:
-                # runs the scroll operation with the request amount of sleep time
-                # for the scroll operation (important to guarantee visibility)
-                self.scroll_to(element, sleep = scroll_sleep)
+                # runs the scroll operation (important to guarantee visibility)

```

Comment:
Tbh I wanted to remove this part too and just let the interactions perform the scroll. Shouldn't be a problem given this is an internal API, right?

In any case, I'm running the tests without this to see what happens.

Reply:
Confirmed: this is not needed.